bin/ares-device.js

```diff
@@ -153,7 +153,7 @@ function finish(err, value) {
         }
         cliControl.end(-1);
     } else {
-        log.info('finish():', value);
+        log.info('finish()', value);
```

Comment:
We decided to split this into two parts, info and verbose.

Reply:
Done.